CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 ## v1.2.5 / 2021-04-28
+
 ### 🐛 Fixes
-- [#24](https://github.com/Paubox/paubox-node/pull/24) Removing security dependencies. Implementing a Changelog. (@niwong)
\ No newline at end of file
+
+- [#24](https://github.com/Paubox/paubox-node/pull/24) Removing security dependencies. Implementing a Changelog. (@niwong)
```

Comment:
I didn't yet add anything here. I assume you'd only add them here when you create a new release on GitHub?